bioconda_utils/bioconda_utils-requirements.txt

```diff
@@ -1,7 +1,7 @@
 anaconda-client=1.6.*
 argh=0.26.*
 beautifulsoup4=4.6.*
-conda=4.3.32
+conda=4.3.31
```

Comment:
Could be that https://github.com/conda-forge/conda-feedstock/pull/36 is only waiting on Travis CI..

Reply:
Indeed.